Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/RayTracing/RayTracingSceneUtils.azsli

```diff
@@ -7,11 +7,12 @@
  */
 
 // returns the normalized camera view ray into the scene for this raytracing dispatch thread
-float3 GetViewRayDirection(float4x4 viewProjectionInverseMatrix)
+float3 GetViewRayDirection(float4x4 projectionInverseMatrix, float4x4 viewInverseMatrix)
```

Comment:
Any reason why add an extra mul?

Reply:
Transforming the direction vector to world space, so I needed to adjust the w-component.